web/impact/impact/v1/helpers/profile_helper.py

```diff
@@ -205,12 +205,20 @@ def object_id_field(klass):
                  EXPERT_OBJECT_ID_FIELD))
 
 EXPERT_INDUSTRY_FIELD = {
-    "json-schema": INDUSTRY_TYPE,
+    "json-schema": MPTT_TYPE,
     "GET": {"description": EXPERT_DESCRIPTION,
             "included": COULD_BE_EXPERT_CHECK}}
 
 EXPERT_INDUSTRY_ARRAY_FIELD = {
-    "json-schema": json_array(INDUSTRY_TYPE),
+    "json-schema": json_array(MPTT_TYPE),
+    "GET": {
+        "included": COULD_BE_EXPERT_CHECK,
+        "description": EXPERT_DESCRIPTION,
```

Comment:
should we collapse EXPERT_INDUSTRY_ARRAY_FIELD and FUNCTIONAL_EXPERTISE_ARRAY_FIELD into one since they are the same?

Reply:
Seems reasonable to me.  Maybe MPTT_ARRAY_FIELD.